azurerm/internal/services/netapp/data_source_netapp_volume.go

```diff
@@ -61,6 +62,18 @@ func dataSourceArmNetAppVolume() *schema.Resource {
 				Type:     schema.TypeInt,
 				Computed: true,
 			},
+
+			"protocols": {
+				Type:     schema.TypeList,
+				Computed: true,
+				MaxItems: 2,
```

Comment:
this is a data source so we don't need the max items

Reply:
Fixed,